docusaurus/docs/nodejs-builtin-fallbacks.md

```diff
@@ -0,0 +1,66 @@
+---
+id: nodejs-builtin-fallbacks
+title: NodeJS builtin fallbacks
+---
+
+NodeJS builtin fallbacks enable you to import NodeJS builtin modules meant for Node and fallback to browser specific modules in your web application.
+
+Per default Create React App set fallbacks to empty modules in production build and development fallbacks in development mode.
```

Comment:
```suggestion
This restores the previous behavior in Webpack < 5, where Create React App used to include these builtin fallbacks.
```

I think this is how the sentence was intended, connecting to the previous sentence. I changed the inclusion to past tense.